app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -136,7 +137,8 @@ public void onFailure(@NonNull Exception e) {
         }
     }
 
-    private void accessFirestoreUsersCollectionForUpdate() {
+    @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE) // making this method always public for testing and private otherwise
+    public void accessFirestoreUsersCollectionForUpdate() {
```

Comment:
This seems like something that could use the backend abstraction

Reply:
Yes that's right. Mark was responsible for implementing the backend abstraction in EditProfileActivity. This should be done in his branch normally!